content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -143,27 +151,41 @@ is recommended to use
 [taints on nodes](/docs/concepts/configuration/taint-and-toleration/)
 to schedule those pods onto the right nodes.
 
-## PodSecurityPolicy Annotations
+## PodSecurityPolicy
 
-The use of sysctl in pods can be controlled via annotation on the PodSecurityPolicy.
+The use of sysctl in pods can be controlled through `allowedUnsafeSysctls` and
+`forbiddenSysctls` fields on the PodSecurityPolicy.
 
-Sysctl annotation represents a whitelist of allowed safe and unsafe sysctls
-in a pod spec. It's a comma-separated list of plain sysctl names or sysctl patterns
-(which end in `*`). The string `*` matches all sysctls.
+By default, all safe sysctls are allowed. Currently, the whitelist of safe
+sysctls corresponds to the safe set mentioned above.
 
-Here is an example, it authorizes binding user creating pod with corresponding sysctls.
+Both `allowedUnsafeSysctls` and `forbiddenSysctls` are lists of plain sysctl names
+or sysctl patterns (which end in `*`). The string `*` matches all sysctls.
+
+The `allowedUnsafeSysctls` field excludes sysctls from the whitelist (`*` means
+no safe sysctls allowed).
```

Comment:
> `*` means no safe sysctls allowed

This is confusing... If there is a `foo` in this field, I'm expecting `foo` as an unsafe sysctl is allowed. If there is a `*` in the list, I'm expecting that all *unsafe* sysctls are allowed. This has nothing to do with *safe* sysctls, right?

Reply:
Right, `allowedUnsafeSysctls` <-> `forbiddenSysctls`. Thanks for noticing that.

Reply:
Fixed.